src/python/WMCore/MicroService/Unified/MSCore.py

```diff
@@ -76,8 +76,9 @@ def updateTransferInfo(self, requestStatuses):
         "Update transfer ids in backend"
```

Comment:
Valentin, if this method is updating a transfer document, then I'd expect it not to get fresh new documents; but instead, either receive an updated document or a list of them.

Reply:
In addition to that, this method is only used by MSMonitor, so we better move it there.

Reply:
Alan, apart from moving it from MSCore to MSMonitor (since MSTransferor now don't use this method) what your first comment asked to do? The method get list of documents and post them to Couch. The input list is passed from execute method where it gets list of new transfer records for given request. Please elaborate what you actually want to do with this method. I don't see anything to be done except moving method from MSCore to MSMonitor.